Samples/azure-storage/Azure.Python/setup.py

```diff
@@ -15,7 +15,7 @@
 # prerequisite: setuptools
 # http://pypi.python.org/pypi/setuptools
 
-REQUIRES = ["msrest>=0.1.0", "msrestazure>=0.1.0"]
```

Comment:
It appears as though whoever checked in python changes didn't re-run regenerate:expected. I am modifying these files as a result of running that after a sync and build.